spec/powergrid.spec.js

```diff
@@ -0,0 +1,121 @@
+var powergrid = require('../powergrid.umd');
+
+
+var config = {
+	name: 'Power Grid',
+	version: '0.1.0',
+	url: 'https://github.com/bblocks/powergrid/',
+	cols: ['minmax(max-content,1fr)', '1fr', '100px', '100vw'],
+	rows: ['minmax(max-content,1fr)', 'minmax(max-content,1fr)', 'minmax(max-content,1fr)'],
+	cells: [
+		{
+			text: '1231231231'
+		},
+		{
+		},
+		{
+		},
+		{
+		},
+		{
+			col: 1,
+			colSpan: 4
+		},
+		{
+		},
+		{
+			row: 2,
+			rowSpan: 2,
+			order: 1
+		},
+		{
+		},
+		{
+		},
+		{
+		},
+		{
+		},
+		{
+		}
+	],
+	prefix: 'grid',
+};
+
+describe('powegrid', function () {
```

Comment:
Unit testing is tricky here since we can really validate the resulting CSS. Or it becomes a way more complex story.